back/test/model_tests/test_crud.py

```diff
@@ -19,29 +20,23 @@ def test_create_qr_code_for_nonexisting_box():
     assert nr_qr_codes == len(QrCode.select())
 
 
-def test_create_box_with_insufficient_data():
```

Comment:
I suppose if you add this test back, codecov stops complaining :smile: @leo-labs

Reply:
I was already on it 🚀